doc/Type/Cool.pod6

```diff
@@ -947,7 +947,8 @@ Defined as:
 Returns the L<unicode property|http://userguide.icu-project.org/strings/properties>
 of the first character. If no property is specified returns the
 L<General Category|https://en.wikipedia.org/wiki/Unicode_character_property#General_Category>.
-Returns a Bool for Boolean properties.
+Returns a Bool for Boolean properties. A L<uniprops|/routine/uniprops> routine can be used
+to get property for every character in a string.
```

Comment:
Missing article: "to get _the_ property"?

Reply:
Thanks!